### DISCUSSION TITLE
how and where is data stored on revanced?

### DISCUSSION THREAD

**Discussion by 0xEthamin** [ReVanced/revanced-discussions#1054] [Question]
Hi, there, 
I'm looking to dump youtube data from revanced app, playlist, subscription etc.
I guess revanced has to store this kind of info locally ? at least some kind of cache ?

Where are they stored and in what format? Is there a way to read them?
the phone is root

thank you for any reply

**Comment by oSumAtrIX** [comment]
revanced does not change anything in this regard to YouTube. It is in the same place as on the unpatched app.
